index.coffee

```diff
@@ -278,6 +299,16 @@ exports.DockerProgress = class DockerProgress
 					console.warn('Progress error:', err.message ? err)
 					totalSize = null
 
+	# Builds and returns a Docker-like progress bar like this:
+	# [==================================>               ] XXX.XX MB/178.83 MB - 64%
+	renderProgress: (percentage, completedSize, totalSize, width = PROGRESS_BAR_WIDTH) ->
+		barCount =  width * percentage // 100
+		spaceCount = width - barCount
+		percentage = prettyPercentage(percentage)
+		bar = "[#{_.repeat('=', barCount)}>#{_.repeat(' ', spaceCount)}]"
```

Comment:
Yeah. Width refers to the scale of the actual progress bar, i.e. how many steps it should have. I don't see a reason to take the rest of the into account when specifying the variable.